game/platform.cc

```diff
@@ -0,0 +1,27 @@
+#include "platform.hh"
+#include "fs.hh"
+
+Platform::platforms Platform::currentOS() {
+if (BOOST_OS_WINDOWS != 0) { return windows; }
+else if (BOOST_OS_LINUX != 0) { return linux; }
+else if (BOOST_OS_MACOS != 0) { return macos; }
+else if (BOOST_OS_BSD != 0) { return bsd; }
+else if (BOOST_OS_SOLARIS != 0) { return solaris; }
+else if (BOOST_OS_UNIX != 0) { return unix; }
+}
```

Comment:
boost also mentions amigaOS, IRIX or IOS. would it be worth it to also check for those? or should we add them in when needed?

Reply:
Maybe. I based those on the platforms that were already implemented in the execname.cc. If you think there might be important differences for one or another (and on which performous would actually run), certainly we could add them.